Flaky test: test_multiple_disks/test.py::test_kill_while_insert

_Important: This issue was automatically generated and is used by CI for matching failures. DO NOT modify the body content. DO NOT remove labels._

Test name: test_multiple_disks/test.py::test_kill_while_insert
Failure reason: E   assert
CI report: [Integration tests (arm_binary, distributed plan, 4/4)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=94582&sha=5de00d60bb676c08edb5bd79991abf14aff9878c&name_0=PR&name_1=Integration%20tests%20%28arm_binary%2C%20distributed%20plan%2C%204%2F4%29)

Failing test history: [cidb](https://play.clickhouse.com#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X211bHRpcGxlX2Rpc2tzL3Rlc3QucHk6OnRlc3Rfa2lsbF93aGlsZV9pbnNlcnQnCiAgICAtLSBBTkQgY2hlY2tfbmFtZSA9ICdJbnRlZ3JhdGlvbiB0ZXN0cyAoYXJtX2JpbmFyeSwgZGlzdHJpYnV0ZWQgcGxhbiwgNC80KScKICAgIEFORCB0ZXN0X3N0YXR1cyBJTiAoJ0ZBSUwnLCAnRVJST1InKQogICAgQU5EIChwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBPUiBiYXNlX3JlZiBJTiAoJ21hc3RlcicpKQogICAgQU5EIHRlc3RfY29udGV4dF9yYXcgTElLRSAnJUUgICBhc3NlcnQlJwpHUk9VUCBCWSBkYXkKT1JERVIgQlkgZGF5IERFU0MK)

Test output:
```
File: test_multiple_disks/test.py:1673 - in test_kill_while_insert
    assert time.time() - start_time < 2
E   assert (1768899628.8605845 - 1768899626.7927125) < 2
E    +  where 1768899628.8605845 = <built-in function time>()
E    +    where <built-in function time> = time.time
```

--- a/tests/integration/test_multiple_disks/test.py
+++ b/tests/integration/test_multiple_disks/test.py
@@ -1662,21 +1662,21 @@ def test_kill_while_insert(start_cluster):
         start_time = time.time()
         long_select = threading.Thread(
             target=ignore_exceptions,
-            args=(node1.query, "SELECT sleep(3) FROM {name}".format(name=name)),
+            args=(node1.query, "SELECT sleep(30) FROM {name}".format(name=name)),
         )
         long_select.start()
 
         sleep_start_time = time.time()
-        time.sleep(0.5)
+        time.sleep(5)
         # long SELECT query might have finished if sleep was too long
-        assert time.time() - sleep_start_time < 1.5
+        assert time.time() - sleep_start_time < 15
 
         node1.query(
             "ALTER TABLE {name} MOVE PARTITION tuple() TO DISK 'external'".format(
                 name=name
             )
         )
-        assert time.time() - start_time < 2.5
+        assert time.time() - start_time < 25
         node1.restart_clickhouse(kill=True)
 
         try:

--- a/tests/integration/test_multiple_disks/test.py
+++ b/tests/integration/test_multiple_disks/test.py
@@ -1676,7 +1676,7 @@ def test_kill_while_insert(start_cluster):
                 name=name
             )
         )
-        assert time.time() - start_time < 2
+        assert time.time() - start_time < 2.5
         node1.restart_clickhouse(kill=True)
 
         try:

